### ISSUE TITLE
Feature Request: Balance Overview Page

### ISSUE THREAD

**Issue by wehmoen** [MattyIce/steem-keychain#108]
Hi,
it would be cool to have overview page that shows
account name, steem, sbd, pending rewards

A "claim all rewards" button would also be awesome especially for people managing many accounts

**Comment by stoodkev** [comment]
Yes it has been planned for a while and I ll start working on it.
